### PR TITLE
Feature/alice connector

### PR DESCRIPTION
**Pull request by yoptar**
Add a `riseapi` mode for simple interaction with Yandex.Dialogs  

alice ner

### CODE REVIEW

**Comment by yoptar** [comment]
@seliverstov 
> `session_id`	Уникальный идентификатор сессии, максимум 64 символов
> `skill_id` Идентификатор вызываемого навыка, присвоенный при создании.
> `user_id` Идентификатор экземпляра приложения, в котором пользователь общается с Алисой, максимум 64 символа  

https://tech.yandex.ru/dialogs/alice/doc/protocol-docpage/

**Comment by yoptar** [comment]
`python -m deeppavlov riseapi --api-mode alice squad_ru`

File: docs/devguides/yandex_dialogs.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Yandex.Dialogs integration
```
Comment (yoptar) [inline docs/devguides/yandex_dialogs.rst]: https://tech.yandex.ru/dialogs/
Yandex.Dialogs is a platform that also includes business chats and we only work with Alice.
Sorry for misguiding you on that before.

File: docs/devguides/yandex_dialogs.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+==========================
+
+
+Different parts of DeepPavlov library could be launched as an endpoint for Yandex.Dialog platform.
```
Comment (yoptar) [inline docs/devguides/yandex_dialogs.rst]: I think we should also include a link on [yandex documentation on that](https://tech.yandex.ru/dialogs/alice/doc/about-docpage/)

File: docs/devguides/yandex_dialogs.rst (hunk 1 above)
Comment (nikolay-bushkov) [reply docs/devguides/yandex_dialogs.rst]: I looked quickly at the documentation, and endpoints for business chats seem to have the same format...

File: docs/devguides/yandex_dialogs.rst (hunk 2 above)
Comment (nikolay-bushkov) [reply docs/devguides/yandex_dialogs.rst]: You can commit anything you want - I am ok with that. I'll not be able to make any changes before tomorrow afternoon.

**Pull request merged**